lib/DriverTool/autolink_extract_main.cpp

```diff
@@ -245,7 +275,7 @@ int autolink_extract_main(ArrayRef<const char *> Args, const char *Argv0,
     }
 
     if (extractLinkerFlags(BinaryOwner->getBinary(), Instance, BinaryFileName,
-                           LinkerFlags)) {
+                           LinkerFlags, CommonAddedSwiftLibraries)) {
```

Comment:
Perhaps it makes sense to sink this entirely into the 3 instances, and use a `constexpr` as an initializer?